### ISSUE TITLE
‘chloropleth’ API endpoint

### ISSUE THREAD

**Issue by loarie** [inaturalist/iNaturalistAPI#56]
endpoint that takes taxon_id and place_id array (as well as other obs search params, e.g.
```api.inaturalist.org/v1/observations/choropleth?places=14,12&taxon_id=2041&verifiable=true…```
and returns observation counts by places in the array. All I need at the moment is place_id, obs_count, e.g.:
```
{
  "total_results": 357,
  "page": 1,
  "per_page": 357,
  "results": {
    "places": {
      "13": 7,
      "15": 30,
      "16": 41,
      "18": 52,
      "31": 97,
      "54": 90,
      ...
    }
  }
}
```
but it might be wise to include the place object in the response if we wanted to anticipate other uses, e.g.:
```
{
  "total_results": 494,
  "page": 1,
  "per_page": 494,
  "results": [
    {
      "place": {
           name: "California",
           id: 14,
           ...
      },
      "observations_count": 10,
    },
    ...
  ]
}
```

**Comment by JWCook** [comment]
I would find this useful as well!

**Comment by jotamayo97** [comment]
Hey! Just wondering if this is still useful. I'd be happy to code it, but not sure if it's still needed or if something similar has already been built and I just didn’t spot it. 
If you still need it, I’m claiming it!

**Comment by kueda** [comment]
Wow, this is an old one. I certainly don't object to it. @pleary any opinions? IMO the endpoint should be `/observations/places` or `/observations/place_counts` to match similar endpoints and so the name describes the resource being returned, not the kind of visualization you might make with the data. Also, the response should match similar endpoints, so something more like

```json
{
  "total_results": 2,
  "page": 1,
  "per_page": 500,
  "results": [
    {
      "count": 140,
      "place": {
        "id": 1234
      }
    },
    {
      "count": 126,
      "place": {
        "id": 4321
      }
    }
  ]
}
```

It should take all the params that the other observations endpoints take, but I think to achieve what Scott asked for we would need an additional param to specify which place aggregations to return, b/c place_id will just select observations in those places, not specify that the requestor only wants to see counts for those places. `place` is a bit vague, maybe `count_place_id` or `group_place_id`?

**Comment by pleary** [comment]
Yes this all sounds great with @kueda's suggestions. No we haven't implemented an equivalent of this yet, so this is free to work on.
